actionlist/app/components/action_item.js

```diff
@@ -0,0 +1,82 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
This directive shouldn't be needed with the change to the global linting rules on `develop`.